src/main/java/com/github/fishio/control/SinglePlayerController.java

```diff
@@ -258,9 +258,6 @@ public void revive() {
 		ICollisionArea area = pf.getStartCollisionArea();
 		player.setBoundingArea(area);
 		
-		//Reset the speed of the fish.
-		player.setSpeedVector(new Vec2d(0, 0));
-		
```

Comment:
When reviving, the player is supposed to reappear in the middle of the screen and lose its speed. With this removed, are you sure that still happens?

<review